nwg_panel/modules/sni_system_tray/tray.py

```diff
@@ -29,37 +29,15 @@ def resize_pix_buf(image, pixbuf, icon_size):
         factor = icon_size / h
     pixbuf = pixbuf.scale_simple(w * factor, h * factor, GdkPixbuf.InterpType.BILINEAR)
 
-    image.set_from_pixbuf(pixbuf)
-
-
-def load_icon(image, icon_name: str, icon_size, icons_path=""):
-    icon_theme = Gtk.IconTheme.get_default()
-    search_path = icon_theme.get_search_path()
-    try:
-        if icons_path:
-            search_path.append(icons_path)
-            icon_theme.set_search_path(search_path)
```

Comment:
`load_icon` did almost the same as `create_pixbuf` except for this search path stuff. `set_search_path` is added to `create_pixbuf` now, so we simply call it rather than duplicate the logic.